src/updatev.jl

```diff
@@ -6,9 +6,12 @@
 Update the noise variances `M.v` with respect to data `Y` using `method`.
 Internally calls [`updatevl(_)`](@ref) to update each entry.
 """
-function updatev!(M::HePPCATModel,Y,method)
+function updatev!(M::HePPCATModel,Y,method, eps)
```

Comment:
Rather than modify this function, I'd prefer we add a new option for `method`,
e.g., as follows
```julia
struct ProjectedVariance{S,T}
    method::S
    varfloor::T
end

updatevl(vl,U,λ,Yl,method::ProjectedVariance) = max(
    updatevl(vl,U,λ,Yl,method.method),
    method.varfloor
)
```

Then in `heppcat` we simply change
```julia
updatev!(M,Y,ExpectationMaximization())
```
to be
```julia
updatev!(M,Y,ProjectedVariance(ExpectationMaximization(),varfloor))
```

Reply:
ok, fancy!